generators/server/templates/src/test/java/_package_/security/DomainUserDetailsServiceIT.java.ejs

```diff
@@ -133,7 +133,34 @@ class DomainUserDetailsServiceIT {
 <%_ if (databaseTypeSql && reactive) { _%>
         userThree.setCreatedBy(Constants.SYSTEM);
 <%_ } _%>
-        userRepository.save(userThree)<% if (reactive) { %>.block()<% } %>;
+        return userThree;
+    }
+
+    @BeforeEach
+    public void init() {
+        userRepository.save(getUserOne())<%- callBlock %>;
+
+        userRepository.save(getUserTwo())<%- callBlock %>;
+
+        userRepository.save(getUserThree())<%- callBlock %>;
+    }
```

Comment:
I'd remove the blank lines here. I often find myself tightening up the tests and removing blank lines like these in my tutorials and book. If prettier does this automatically, please ignore.